README.md

```diff
@@ -58,19 +59,17 @@ Connect will instantiate the MarkLogic Kafka connector based on the configuratio
 
 ## Configuring the connector
 
-Using the MarkLogic Kafka connector requires configuring a set of properties to control how Kafka passes data to 
-the connector, how the connector connects to MarkLogic, and how data is written to MarkLogic. 
-
-The manner in which you use the MarkLogic connector will determine how you configure the connector:
+Using the MarkLogic Kafka connector requires configuring a set of properties to control how the connector interacts 
+with MarkLogic. The manner in which you use the MarkLogic connector will determine how you configure the connector:
 
 - If you are using the connector via Confluent Control Center, the Control Center GUI will list all the connector
   properties with inline help showing a description of each property.
 - If you are using the connector by running Kafka Connect from the command line, you'll be configuring a copy of the
-  `./config/marklogic-sink.properties` file in this repository which also lists all the connector properties with a 
-  description for each one.
+  `./config/marklogic-sink.properties` file or `./config/marklogic-source.properties` file in this repository which 
+  lists all the connector properties with a description for each one.
 
-In addition to the properties found in the locations above, you'll also need to configure two Kafka properties that 
-control [how data is serialized](https://www.confluent.io/blog/kafka-connect-deep-dive-converters-serialization-explained/) 
+When using the sink connector, you'll also need to configure two Kafka properties that control
```

Comment:
These likely need to be configured for the source connector, just not certain as to how yet.